multicast/__main__.py

```diff
@@ -601,8 +603,8 @@
 
 if __name__ in '__main__':
 	__EXIT_CODE = (2, "NoOp")
-	if (sys.argv is not None) and (len(sys.argv) > 1):
-		__EXIT_CODE = main(sys.argv[1:])
-	elif (sys.argv is not None):
+	if (_sys.argv is not None) and (len(_sys.argv) > 1):
+		__EXIT_CODE = main(_sys.argv[1:])
+	elif (_sys.argv is not None):
 		__EXIT_CODE = main([str(__name__), """-h"""])
 	exit(__EXIT_CODE[0])
```

Comment:
While I love the idea of storing the exit codes in a dictionary with accompanying messages. However the focus of the __main__.py is dispatching the various sub-commands rather than the exitcodes. This is not to diminish the importance of the exit-codes, just to de-couple the args from exit-codes.